scripts/report-bundle-sizes-as-a-pr-comment.js

```diff
@@ -0,0 +1,189 @@
+const { command } = require('./lib/command')
+const { runMain, timeout } = require('./lib/execution-utils')
+const { getOrg2ApiKey, getGithubAccessToken, getOrg2AppKey } = require('./lib/secrets')
+
+const PR_COMMENT_HEADER = 'Bundles Sizes Evolution'
+const BASE_BRANCH = process.env.MAIN_BRANCH
+const LOCAL_BRANCH = process.env.CI_COMMIT_REF_NAME
+const PR_COMMENTER_AUTH_TOKEN = command`authanywhere`.run().split(' ')[2].trim()
+const GITHUB_TOKEN = getGithubAccessToken()
+const ONE_DAY_IN_SECOND = 24 * 60 * 60
+const TIMEOUT_DURATION_MS = 5000
+const FETCH_RETRIES = 4
+
+runMain(async () => {
+  const lastCommonCommit = getLastCommonCommit(BASE_BRANCH, LOCAL_BRANCH)
+  const latestLocalCommit = process.env.CI_COMMIT_SHORT_SHA
+  const prs = await getPRs(LOCAL_BRANCH)
+  if (prs.length === 0) {
+    console.log('No pull requests found for the branch')
+    process.exit(0)
+  }
+  if (prs.length > 1) {
+    console.log('Multiple pull requests found for the branch')
+    process.exit(0)
+  }
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏In this case, we should probably have a look so what about failing the script?